package/src/components/CartSummary/v1/CartSummary.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { applyTheme } from "../../../utils";
+
+const Table = styled.table`
+  min-width: 20rem;
```

Comment:
This `min-width` is causing the table to overflow its container on smaller mobile devices. I think you can get a similar layout by removing the `min-width` and adding `width: 100%` and possibly a `max-width` if we need it to cap its width but from the looks of the designs, I think this component should just fill the available space.